jenkins/requirements.txt

```diff
@@ -0,0 +1,9 @@
+setuptools==36.0.1
+pyinstaller==3.5
+PyYAML==5.1.2
+argparse==1.4.0
+dataclasses==0.7
+python-consul==1.1.0
+pathlib==1.0.1
+pytest==5.4.2
+cortx-py-utils>=1.0.0
```

Comment:
cortx-py-utils>=1.0.0

is it python package.

why separate package file needed ; we already have pyinstaller requirement.txt

Reply:
That's inside pyinstaller. Didn't want to refer to that and moving out would have most likely broken RE code. Have tried to keep both pyinstaller and setuptools separate as much as possible and maintain backward compatibility.

Reply:
Removed cortx-py-utils.

Reply:
We should re-visit this later. Since cortx-py-utils gets installed in site-packages, this check might be okay.